hack/install-tools.sh

```diff
@@ -65,15 +73,19 @@ case "$1" in
 		_require_gobin
 		_install_gosec
 		;;
+	--gitlint)
+		_install_gitlint
+		;;
 	*)
 		echo "usage: GOBIN=<dir> $0 --<tool-name>"
 		echo ""
-		echo "availabel tools:"
```

Comment:
typo: availabel

Reply:
> This change fixes that typo

oh, now I see! thanks

Reply:
Thanks! I was starting to wonder if you were trolling me. ;-)